### PR TITLE
Refactor code.

### PR DESCRIPTION
**Pull request by pradeepbhadani**

### CODE REVIEW

File: cloudwatch.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
```
Comment (mroark-exp) [inline cloudwatch.tf]: 2019

File: ecs-service-desicovery.tf [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/**
```
Comment (mroark-exp) [inline ecs-service-desicovery.tf]: file name has a typo

File: cloudwatch.tf (hunk 1 above)
Comment (pradeepbhadani) [reply cloudwatch.tf]: @mroark1m  added

File: ecs-service-desicovery.tf (hunk 2 above)
Comment (pradeepbhadani) [reply ecs-service-desicovery.tf]: fixed :)

**Pull request merged**